contracts/eosio.system/include/eosio.system/native.hpp

```diff
@@ -1,51 +1,87 @@
 /**
- *  @file
  *  @copyright defined in eos/LICENSE.txt
  */
+
 #pragma once
 
-#include <eosiolib/action.hpp>
-#include <eosiolib/public_key.hpp>
-#include <eosiolib/print.hpp>
-#include <eosiolib/privileged.h>
-#include <eosiolib/producer_schedule.hpp>
-#include <eosiolib/contract.hpp>
-#include <eosiolib/ignore.hpp>
-#include <eosiolib/fixed_bytes.hpp>
+#include <eosio/action.hpp>
+#include <eosio/contract.hpp>
+#include <eosio/crypto.hpp>
+#include <eosio/fixed_bytes.hpp>
+#include <eosio/ignore.hpp>
+#include <eosio/print.hpp>
+#include <eosio/privileged.hpp>
+#include <eosio/producer_schedule.hpp>
+
+// extern "C" {
+//    struct __attribute__((aligned (16))) capi_checksum160 { uint8_t hash[20]; };
+//    struct __attribute__((aligned (16))) capi_checksum256 { uint8_t hash[32]; };
+//    struct __attribute__((aligned (16))) capi_checksum512 { uint8_t hash[64]; };
+// }
+
+// namespace eosio {
+//    namespace internal_use_do_not_use {
+//       extern "C" {
+//          __attribute__((eosio_wasm_import))
+//          bool is_feature_activated( const ::capi_checksum256* feature_digest );
+
+//          __attribute__((eosio_wasm_import))
+//          void preactivate_feature( const ::capi_checksum256* feature_digest );
+//       }
+//    }
+// }
+
+// namespace eosio {
+//    bool is_feature_activated( const eosio::checksum256& feature_digest ) {
+//       auto feature_digest_data = feature_digest.extract_as_byte_array();
+//       return internal_use_do_not_use::is_feature_activated(
+//          reinterpret_cast<const ::capi_checksum256*>( feature_digest_data.data() )
+//          );
+//    }
+
+//    void preactivate_feature( const eosio::checksum256& feature_digest ) {
+//       auto feature_digest_data = feature_digest.extract_as_byte_array();
+//       internal_use_do_not_use::preactivate_feature(
+//          reinterpret_cast<const ::capi_checksum256*>( feature_digest_data.data() )
+//          );
+//    }
+// }
 
 namespace eosio {
```

Comment:
Remove the commented out lines above and change this back to `::capi_checksum256` and place a comment above this section stating "needed until these make it into eosio.cdt" and add the include `#include <eosio/../../capi/eosio/crypto.h>`.